git_theta/updates/base.py

```diff
@@ -125,15 +125,19 @@ async def write(
         param: Parameter,
         param_keys,
         *,
-        param_metadata: metadata.ParamMetadata,
+        prev_param_metadata: metadata.ParamMetadata,
         repo,
         path: str,
         **kwargs,
     ) -> metadata.LfsMetadata:
         """Serialize and save a parameter with git-lfs as a delta from the previous value."""
         logging.debug(f"Writing {self.name} update for {'/'.join(param_keys)}")
-        previous_value = await self.get_previous_value(
-            param_metadata, param_keys, repo=repo, path=path
+        prev_update_type = prev_param_metadata.theta_metadata.update_type
```

Comment:
@blester125 this fixes the bug where you need to do a second dense update before doing a sparse update.